apps/remote_control/lib/lexical/remote_control/search/fuzzy.ex

```diff
@@ -359,8 +359,26 @@ defmodule Lexical.RemoteControl.Search.Fuzzy do
         names
       end
 
-    fn mapped(application: app, subtype: subtype) ->
-      subtype == :definition and MapSet.member?(app_names, app)
+    # we do the path thing to ensure that
+    # we get a valid OS-specifc path component
+    # so we can do `String.contains?` on a path
+    deps_part =
+      [" ", "deps", " "]
+      |> Path.join()
+      |> String.trim()
+
+    fn
+      mapped(application: nil, subtype: :definition, grouping_key: path) ->
+        # if we don't have an app name, just make sure we're not
+        # in what looks like a deps directory
+
+        not String.contains?(path, deps_part)
```

Comment:
What happens if our module has `Deps` in it? e.g. we're in `test/my_lib/parent/deps/child_test.exs`

Reply:
it won't show up.

Reply:
Technically, we'd need to interrogate each project's mix config to figure out what directory its deps reside in, `deps` is just the default, and I don't know how to go from app name -> mix project or from a directory to a mix project.

I can try to come up with a 80% heuristic though, that should be no worse than the above.